runtime/runtime/src/lib.rs

```diff
@@ -313,7 +313,7 @@ impl Runtime {
         result.gas_used += exec_fees;
         let account_id = &receipt.receiver_id;
         let is_the_only_action = actions.len() == 1;
-        let is_refund = AccountId::is_system(&receipt.predecessor_id);
+        let is_refund = receipt.predecessor_id.as_ref() == "system";
```

Comment:
what was the reason for removing 'is_system' (can you mention it in PR decription)

Reply:
Done.

Reply:
Hm, `system` account is a protocol-level concept, so it actually seems fine to have one: https://nomicon.io/DataStructures/Account.html?highlight=system#system-account. 

And it is possible to observe system account id from the outside:

https://explorer.testnet.near.org/transactions/BBqGuajZrRBPpiQZMwJyJj1NNR3Q5P9KUD1w7HuCiz7C#HRKLfrnczeCsegJb5gVJzBhiMs7LXLe1Soq8RbqNvVLN

![image](https://user-images.githubusercontent.com/1711539/144448325-35814ddf-75ab-4968-b1d5-ee4d73ed8f6f.png)

So I think `AccountId` should include public `is_system` method for completness, even if it is not very practically useful outside.

Reply:
Understood, I'll restore it.